### PR TITLE
DF 712 - logout missing on mobile app

### PR DESCRIPTION
**Pull request by emina-str**
Preview of logout menu on mobile:
df-712

### CODE REVIEW

**Comment by pavax** [comment]
the alignment of the logout button doesn't match with the other icons

File: alv-portal-ui/src/app/shared/layout/main-navigation/main-navigation.component.html [hunk 1]
```diff
@@ -75,6 +75,16 @@ <h1 class="sr-only">Navigation</h1>
...
 
   </div>
+  <hr>
+  <!--Logout Button-->
+  <a class="nav-item"
+     *alvIsAuthenticated
+     (click)="logout()">
+    <fa-icon class="nav-icon"
+             [fixedWidth]="true"
+             [icon]="['fas', 'power-off']"></fa-icon>
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/shared/layout/main-navigation/main-navigation.component.html]: for login we have:
<fa-icon class="nav-icon mx-2" [icon]="['fas', 'key']"></fa-icon>

Comment (emina-str) [reply alv-portal-ui/src/app/shared/layout/main-navigation/main-navigation.component.html]: but for logout icon should be "power-off" ... also in dummy it looks like other menu-entry items, not like login. https://rlwa8z.axshare.com/#g=1&p=mobil-startseite_jim_jobsucher&c=1

Comment (pavax) [reply alv-portal-ui/src/app/shared/layout/main-navigation/main-navigation.component.html]: what julien meant is that the alignment for the other buttons have been using the 'mx-2'

Comment (emina-str) [reply alv-portal-ui/src/app/shared/layout/main-navigation/main-navigation.component.html]: when usin 'mx-2' it looks like this 
df-712--using mx-2

Comment (cjpsycos) [reply alv-portal-ui/src/app/shared/layout/main-navigation/main-navigation.component.html]: ok

**Pull request merged**